src/subcategories/connection.ts

```diff
@@ -1,7 +1,14 @@
 import { makeEnum } from '@transcend-io/type-utils';
 
+/**
+ * Permanent (e.g., device IDs, MAC address) or temporary (e.g., IP address)
+ * identifiers needed to establish a connection for the current browsing session.
+ */
 export const ConnectionSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** A Media Access Control address */
+  MAC_ADDRESS: 'MAC_ADDRESS',
+  /** An internet protocol address */
+  IP_ADDRESS: 'IP_ADDRESS',
```

Comment:
Just curious - why do MAC_ADDRESS and IP_ADDRESS get all-capitalized keys when the others are camel cased? Later on it is written `MACAddress` and `IPAddress`, so one of those might be an oversight.

Reply:
Def an oversight! Thanks for catching